Packs/RecordedFuture/ReleaseNotes/1_6_3.md

```diff
@@ -0,0 +1,15 @@
+
+#### Integrations
+
+##### Recorded Future - Lists
+
+- Added remove entities from lists command
+- Added command to fetch entities from a list 
```

Comment:
Put the name of the command here.
For example:
Added the xxx command to fetch entities from a list.

Reply:
done